README.md

```diff
@@ -1,2 +1,10 @@
 # XebiaLabs-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: XebiaLabs, Appium and ReactNative.
+
+XebiaLabs is a service that develops Continuous Delivery and DevOps software to provide companies greater control over their software delivery process. In XL Release, you can plan, automate, and optimize your pipeline.
+
+The guidelines in this repo will show you how to plan a release with the XebiaLabs service XL release as well as how to trigger a Jenkins build from a task on XebiaLabs.
+
+## Contents
+ 1. First guild will demonstrate [how to plan a release on XL Release](planning-release.md).
+
+ 2. The second guide will demonstrate [how to trigger a Jenkins build from a task on XL Release](triggering-jenkins.md).
```

Comment:
how to trigger a Jenkins build from a task on XL Release --> how to integrate executing tests on Kobiton devices through triggering a Jenkins job